sample-parent/tck-validator/README.adoc

```diff
@@ -0,0 +1,2 @@
+= A tck sample Family
```

Comment:
purpose?

Reply:
Used in the Test case, discussed with Axel, or, only used in test case, not commit?